habitat/plan.sh

```diff
@@ -168,11 +170,16 @@ do_strip() {
 
 # Copied from https://github.com/habitat-sh/core-plans/blob/f84832de42b300a64f1b38c54d659c4f6d303c95/bundler/plan.sh#L32
 wrap_ruby_bin() {
-  local bin_basename="$1"
-  local real_cmd="$pkg_prefix/ruby-bin/$bin_basename"
-  local wrapper="$pkg_prefix/bin/$bin_basename"
-
-  build_line "Adding wrapper $wrapper for $real_cmd"
+  local bin_basename
+  local real_cmd
+  local wrapper
+  bin_basename="$1"
+  real_cmd="$ruby_bin_dir/$bin_basename"
+  wrapper="$pkg_prefix/bin/$bin_basename"
+
+  build_line "Adding wrapper for $bin_basename."
```

Comment:
Could we do these on one line, eg `Adding wrapper for $bin_basename: $wrapper -> $real_cmd`